test/test_common/utility.h

```diff
@@ -199,44 +199,52 @@ class TestUtility {
   static Stats::GaugeSharedPtr findGauge(Stats::Store& store, const std::string& name);
 
   /**
-   * Wait till Counter value is equal to the passed ion value.
+   * Wait for a counter to == a given value.
```

Comment:
nit: expand equal?

Reply:
I was just copying the format of the other comments, will i change them all?

Reply:
Ah, didn't notice the other ones were like that too (diff eye catching...). Fine as is :)